ts/src/base/functions/number.ts

```diff
@@ -98,12 +99,34 @@ function precisionFromString (str) {
 
 /*  ------------------------------------------------------------------------ */
 
-const decimalToPrecision = (
+const decimalToPrecisionWrapper = (
     x,
     roundingMode,
     numPrecisionDigits,
     countingMode = DECIMAL_PLACES,
     paddingMode = NO_PADDING
+) => {
+    // Turn precision value to and integer
+    let precision = numPrecisionDigits;
+    if (typeof precision === 'string') {
+        precision = parseFloat(precision)
+    }
+    if (countingMode === TICK_SIZE) {
+        const precisionDigitsString = decimalToPrecision (precision, ROUND, 22, DECIMAL_PLACES, NO_PADDING);
+        precision = precisionFromString (precisionDigitsString);
+    }
+    // Eliminate rounding errors
+    x = decimalToPrecision (x, ROUND, precision+2, DECIMAL_PLACES, NO_PADDING);
```

Comment:
For `2` and `22`, could you please put these numbers to constants with clear name? Would be nice to explain why did you choose these numbers in comments